tools/perf/brave_page_sets/brave_system_health/system_health_stories.py

```diff
@@ -104,3 +104,34 @@ class LoadBBCStory2023(_BraveLoadingStory):
   NAME = 'load:site:bbc:2023'
   URL = 'https://www.bbc.com/'
   TAGS = [story_tags.YEAR_2023]
+
+
+class LoadHackernewsStory2023(_BraveLoadingStory):
+  NAME = 'load:site:hackernews:2023'
+  URL = 'https://news.ycombinator.com/'
+  SCROLL_PAGE = False
+  TAGS = [story_tags.YEAR_2023]
+
+
+class LoadPinterestStory2023(_BraveLoadingStory):
+  NAME = 'load:site:pinterest:2023'
+  URL = 'https://www.pinterest.com/search/pins/?q=home%20decor'
+  TAGS = [story_tags.YEAR_2023]
+
+
+class LoadIMDBStory2023(_BraveLoadingStory):
+  NAME = 'load:site:imdb:2023'
+  URL = 'https://www.imdb.com/what-to-watch/popular/'
+  TAGS = [story_tags.YEAR_2023]
+
+
+class LoadAmazonStory2023(_BraveLoadingStory):
+  NAME = 'load:site:amazon:2023'
+  URL = 'https://www.amazon.com/gp/new-releases/'
+  TAGS = [story_tags.YEAR_2023]
+
+
+class LoadBraveNewsStory2023(_BraveLoadingStory):
+  NAME = 'load:ntp:brave_news:2023'
+  URL = 'chrome://newtab/'
+  TAGS = [story_tags.YEAR_2023]
```

Comment:
why 2023 is used everywhere? What's the point of this tag?

Reply:
Ah, it's chromium originated : https://source.chromium.org/chromium/chromium/src/+/main:tools/perf/page_sets/system_health/story_tags.py;l=66?q=YEAR_2023
Initially, I added 2024, but it cost us +1 patch. Do you thing it makes sense?

Reply:
can we just not use the tag?

Reply:
It requires some non-trivial changes in system_health. Now the tests assume that all the story in the same format.
That approach is also easy to be broken in chrome major bumps.